settings.html

```diff
@@ -49,10 +49,20 @@ <h2 style="font-family: Gill Sans Light">Show News Recommendation</h2>
                     <h4>Show Web News -> TV News Recommendations</h4>
                     <label for="news" style="font-size: 14px;">
                         <input type="checkbox" id="news">
-                        <span class="enable"><b>Present "News-Recommendation" pop-ups</b></span> 
+                        <span class="enable"><b>Present "News-Recommendation" pop-ups</b></span>
                     </label>
                 </div>
             </div>
+						<div id="show_option_wikibooks"  class="col-sm-3">
+								<div class="wikibooks" style="padding-top: 5px;">
+										<h2 style="font-family: Gill Sans Light">Wikipedia books settings</h2>
+										<h4>Read books cited by Wikipedia</h4>
+										<label for="wikibooks" style="font-size: 14px;">
+												<input type="checkbox" id="wikibooks">
+												<span class="enable"><b>Present "Wikibooks" pop-ups</b></span>
+										</label>
+								</div>
+						</div>
```

Comment:
Please watch out the code identation, it should be uniform in all the code. Please fix this.

Reply:
That's weird.  It is uniform in my IDE.  
![image](https://user-images.githubusercontent.com/7968828/45564271-c9cf8300-b804-11e8-8a59-2862cf86d50d.png)